courses/models.py

```diff
@@ -524,3 +526,164 @@ class ProgramEnrollmentAudit(AuditModel):
     @classmethod
     def get_related_field_name(cls):
         return "enrollment"
+
+
+class FinalGradeQuerySet(models.QuerySet):
+    """
+    QuerySet class defining common query parameters for FinalGrade
+    """
+    def passed(self):
+        """
+        Returns a queryset with a filter that will only fetch passed course runs
+        """
+        return self.filter(passed=True)
+
+    def paid_on_edx(self):
+        """
+        Returns a queryset with a filter that will only fetch course runs that are paid on edX.
+        Note that a course run can still be paid via an Order if this value is False for a given user.
+        """
+        return self.filter(course_run_paid_on_edx=True)
```

Comment:
Again, we don't have this dependency on edX. If the user has an enrollment, then they have paid.